build.py

```diff
@@ -118,35 +130,60 @@
                                 f.write(res.stderr.decode(stdout_err_encoding))
 
 
-# When using subprocess there is no nice way to try (short of threads) to capture the output and send
-# it to the console while also capturing it to a file.  So we use sh instead.
-# https://amoffat.github.io/sh/index.html
-# sh "knows" about all of the commands on your PATH
+# Build Docker containers
```

Comment:
I really think the home for this, as I suggested in Discord is in the `components/rsptx/cl_utils/core.py` file.

I would suggest augmenting the run_command with an optional function that can take the place of the output_stream function.  This makes it nearly functionally equivalent to what we had with sh